packages/devtools_app/lib/src/screens/debugger/codeview_controller.dart

```diff
@@ -50,8 +51,22 @@ class CodeViewController extends DisposableController
         {
           final processedState =
               CodeViewSourceLocationNavigationState._fromState(state);
-          // TODO(bkonyi): investigate delay in scrolling to source location.
+          final object = processedState.object;
```

Comment:
Consider pulling body of the `case` clause into a separate method for clarity, eg. `_updateCodeViewSource`